bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -201,6 +201,18 @@ <h3 data-accordion-role="tab">{{ _('How do I turn off Location-Aware Browsing pe
   </div>
 </section>
 
+<section>
+  <h3 data-accordion-role="tab">{{ _('How can I add support for Geolocation to my web site?') }}</h3>
+  <div data-accordion-role="tabpanel">
+    <p>
+    {% trans url="https://developer.mozilla.org/en/Using_geolocation" %}
```

Comment:
This URL now redirects to a newer page, so you can change this link to: `https://developer.mozilla.org/docs/Web/API/Geolocation/Using_geolocation`. 

Note I have left the `en-US` part of the URL out here, which allows MDN to best decide the correct language for whoever clicks the link :)